### ISSUE TITLE
Filter failed to return unique result

### ISSUE THREAD

**Issue by rnliseb** [cdhayward/Enable-TeamsDRUser#1]
I usually have no issues with this script, however when assigning a phone number that was previously assigned to someone else, I got this error during the "enabling enterprise voice step": 

`Filter failed to return unique result, "[SipAddress : sip:user@domain.com] [LineURI : tel:+xxxxxxxxxxxx]
[PrivateLine : tel:+xxxxxxxxxxxx] "
    + CategoryInfo          : InvalidOperation: (Set-CsUser:String) [Set-CsUser], InvalidOperationException
    + FullyQualifiedErrorId : ProcessRecord,Microsoft.Rtc.Management.AD.Cmdlets.SetOcsUserCmdlet
    + PSComputerName        : api.interfaces.records.teams.microsoft.com`

I ran the powershell manually, and it applied no problem:
Set-CsUser -Identity user@domain.com -OnPremLineURI tel:+xxxxxxxxxxxx -EnterpriseVoiceEnabled $true

**Comment by cdhayward** [comment]
Hi the "Filter failed to return unique result" is a Teams error and is most likely just the Teams backend being slow after removing the original number.

The script is running the same Set-CsUser command (Line 410) `Set-CsUser -Identity $SipAddress -OnPremLineURI $txtb_LineURI.text -EnterpriseVoiceEnabled $true`

**Comment by rnliseb** [comment]
Yeah I saw that the script runs the exact same thing, strange - I did wait 24 hours after removing the number and tried both ways just after another
